### PR TITLE
Decomp gf_3d_vrammanager; finish field_follow_poke

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/field_follow_poke.c [hunk 1]
```diff
@@ -203,3 +1684,351 @@ void sub_02069DEC(LocalMapObject *mapObject, int a1) {
...
+BOOL FollowingPokemon_IsActive(FieldSystem *fsys) {
+    if (fsys->followMon.active == 0) {
+        return FALSE;
+    }
+
+    if (CountAlivePokemon(SavArray_PlayerParty_get(fsys->savedata)) != 0) {
+        return TRUE;
+    } else {
+        return FALSE;
+    }
```
Comment (tgsm) [inline src/field_follow_poke.c:1786]:
```suggestion
    return CountAlivePokemon(SavArray_PlayerParty_get(fsys->savedata)) != 0;
```

File: src/field_follow_poke.c [hunk 2]
```diff
@@ -203,3 +1684,351 @@ void sub_02069DEC(LocalMapObject *mapObject, int a1) {
...
+BOOL sub_02069FB0(FieldSystem *fsys) {
+    if (FollowingPokemon_IsActive(fsys)) {
+        if (!sub_0205F684(fsys->followMon.mapObject)) {
+            return TRUE;
+        } else {
+            return FALSE;
+        }
+    } else {
+        return FALSE;
+    }
```
Comment (tgsm) [inline src/field_follow_poke.c:1798]:
```suggestion
    if (!FollowingPokemon_IsActive(fsys)) {
        return FALSE;
    }

    return !sub_0205F684(fsys->followMon.mapObject);
```

**Review by tgsm — CHANGES_REQUESTED** [review]

File: src/field_follow_poke.c (hunk 1 above)
Comment (PikalaxALT) [reply src/field_follow_poke.c:1786]: Stylistically inconsistent

File: src/field_follow_poke.c (hunk 2 above)
Comment (tgsm) [reply src/field_follow_poke.c:1798]: That's odd. Maybe there's an issue on my end

**Review by tgsm — APPROVED** [review]

**Pull request merged**